vignettes/spatial_wwinference.Rmd

```diff
@@ -0,0 +1,2457 @@
+  ) +
+  geom_line(
+    data = hosp_ribbon_data,
+    aes(x = date, y = median, color = inf_model_type),
+    size = 1,
+  ) +
+  geom_point(
+    data = hosp_data_eval_full %>%
+      mutate(gen_model_type = model_type),
+    aes(x = date, y = daily_hosp_admits_for_eval),
+    shape = 21, color = "black", fill = "white"
+  ) +
+  geom_point(
+    aes(x = date, y = observed_value)
+  ) +
+  geom_vline(aes(xintercept = lubridate::ymd(forecast_date)),
+    linetype = "dashed"
+  ) +
+  xlab("") +
+  ylab("Daily hospital admissions") +
+  facet_wrap(~gen_model_type, scales = "free_y") +
+  guides(
+    fill = guide_legend(
+      title = "Assumed Corr. Structure"
+    ),
+    color = guide_legend(
+      title = "Assumed Corr. Structure"
+    )
+  ) +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  theme_bw()
+# ------------------------------------------------------------------------------
+# Wastewater results -----------------------------------------------------------
+ww_ribbon_data <- all_pred_draws_df %>%
+  filter(
+    name == "predicted wastewater"
+  ) %>%
+  mutate(
+    site_lab_name = glue::glue("{subpop}, Lab: {lab}")
+  ) %>%
+  group_by(
+    date,
+    subpop,
+    inf_model_type,
+    gen_model_type
+  ) %>%
+  summarise(
+    lower = quantile(pred_value, 0.025, na.rm = TRUE),
+    median = median(pred_value),
+    upper = quantile(pred_value, 0.975, na.rm = TRUE),
+    .groups = "drop"
+  )
+ww_result_plot <- ggplot(
+  all_pred_draws_df %>%
+    filter(
+      name == "predicted wastewater"
+    )
+) +
+  geom_ribbon(
+    data = ww_ribbon_data,
+    aes(x = date, ymin = lower, ymax = upper, fill = inf_model_type),
+    alpha = 0.1
+  ) +
+  geom_line(
+    data = ww_ribbon_data,
+    aes(x = date, y = median, color = inf_model_type),
+    size = 1,
+  ) +
+  geom_point(
+    aes(x = date, y = observed_value)
+  ) +
+  geom_vline(aes(xintercept = lubridate::ymd(forecast_date)),
+    linetype = "dashed"
+  ) +
+  xlab("") +
+  ylab("Genome copies/mL on Log Scale") +
+  facet_grid(subpop ~ gen_model_type, scales = "free_y") +
+  guides(
+    fill = guide_legend(
+      title = "Assumed Corr. Structure"
+    ),
+    color = guide_legend(
+      title = "Assumed Corr. Structure"
+    )
+  ) +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  theme_bw()
+# ------------------------------------------------------------------------------
+
+(hosp_result_plot / ww_result_plot) +
+  plot_layout(guides = "collect") &
+  theme(legend.position = "bottom")
+```
```

Comment:
Could add some language on any differences in forecast performance you see across inference models or data generating models (where the data generating model differences are likely due to the forecasting problems being different degrees of difficulty).